_1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java

```diff
@@ -0,0 +1,37 @@
+package code._4_student_effort;
+
+public class FizzBuzz {
+
+    public static void main (String[] args){ fizzBuzz(100);}
```

Comment:
daca ar fi sa crestem numarul de la 100, la 110, atunci numarul 105 este divizibil si cu 3, 5, 7 si ar trebui sa afiseze FizzBuzzRizz

cum ai proceda in cazul asta ?

Reply:
Pentru acest caz, in particular, as introduce urmatoarele linii  in primul ciclu for : 
else if((i % 3 == 0) && (i % 5 == 0) && (i % 7 ==0)
       System.out.println("FizzBuzzRizz");

Cand am inceput sa fac challenge-ul pentru FooBarQix mi-am dat seama ca as putea folosi o variabila de tip string careia sa i se atribuiasca un spatiu gol iar apoi prin niste if-uri care verifica divizibilitatea sa i se adauge cuvintele respective.

Reply:
bravo

o sa mai povestim la curs despre termenul de "maintainability"